blocks/top-table-list-block/features/top-table-list/_children/item-title-with-right-image.test.jsx

```diff
@@ -84,153 +59,236 @@ describe('item title with right image block', () => {
-  //     const imageURL = 'pic';
-  //     const itemTitle = 'title';
-  //     const primaryFont = 'arial';
-  //     const id = 'test';
-  //     const { default: ItemTitleWithRightImage } = require('./item-title-with-right-image');
-
-  //     const wrapper = mount(
-  //       <ItemTitleWithRightImage
-  //         imageURL={imageURL}
-  //         itemTitle={itemTitle}
-  //         primaryFont={primaryFont}
-  //         id={id}
-  //         customFields={config}
-  //         resizedImageOptions={{ '400x267': '' }}
-  //       />,
-  //     );
-
-  //     expect(wrapper.find('.small-promo-one').length).toBe(0);
-  //     expect(wrapper.find('article.wrap-bottom').length).toBe(1);
-  //   });
-
-  //   it('when storiesPerRowSM is 2 must not add class small-promo-one', () => {
-  //     const imageURL = 'pic';
-  //     const itemTitle = 'title';
-  //     const primaryFont = 'arial';
-  //     const id = 'test';
-  //     const { default: ItemTitleWithRightImage } = require('./item-title-with-right-image');
-  //     const setup = Object.assign(config, { storiesPerRowSM: 2 });
-
-  //     const wrapper = mount(
-  //       <ItemTitleWithRightImage
-  //         imageURL={imageURL}
-  //         itemTitle={itemTitle}
-  //         primaryFont={primaryFont}
-  //         id={id}
-  //         customFields={setup}
-  //         resizedImageOptions={{ '400x267': '' }}
-  //       />,
-  //     );
-
-  //     expect(wrapper.find('.small-promo-one').length).toBe(0);
-  //     expect(wrapper.find('article.wrap-bottom').length).toBe(1);
-  //   });
-
-  //   it('when storiesPerRowSM is 1 must add class small-promo-one', () => {
-  //     const imageURL = 'pic';
-  //     const itemTitle = 'title';
-  //     const primaryFont = 'arial';
-  //     const id = 'test';
-  //     const { default: ItemTitleWithRightImage } = require('./item-title-with-right-image');
-  //     const setup = Object.assign(config, { storiesPerRowSM: 1 });
-
-  //     const wrapper = mount(
-  //       <ItemTitleWithRightImage
-  //         imageURL={imageURL}
-  //         itemTitle={itemTitle}
-  //         primaryFont={primaryFont}
-  //         id={id}
-  //         customFields={setup}
-  //         resizedImageOptions={{ '400x267': '' }}
-  //       />,
-  //     );
-
-//     expect(wrapper.find('.small-promo-one').length).toBe(1);
-//     expect(wrapper.find('article.wrap-bottom').length).toBe(0);
-//   });
+  it('must render only image if showHeadlineSM false in horizontal layuout', () => {
+    const imageURL = 'pic';
+    const itemTitle = 'title';
+    const primaryFont = 'arial';
+    const id = 'test';
+    const { default: ItemTitleWithRightImage } = require('./item-title-with-right-image');
+
+    const wrapper = mount(
+      <ItemTitleWithRightImage
+        imageURL={imageURL}
+        itemTitle={itemTitle}
+        primaryFont={primaryFont}
+        id={id}
+        /* eslint-disable-next-line */
```

Comment:
we might want to apply this to the whole file if we have it in a few places

Reply:
but that will disable the eslint on the whole file, not sure we want that.
The problem here is that eslint complaint about Object.assign and suggest to use the deconstructor way, but other nasty error happens using the deconstructor.